src/libraries/System.IO.FileSystem/tests/File/Create.cs

```diff
@@ -219,9 +220,13 @@ public void LongFileName()
 
         [Fact]
         [PlatformSpecific(CaseSensitivePlatforms)]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/51371", TestPlatforms.iOS | TestPlatforms.tvOS | TestPlatforms.MacCatalyst)]
```

Comment:
you can apply the same fix to the `SearchPatternCaseSensitive` test which is also disabled against this issue.